components/OutivityForm/index.js

```diff
@@ -56,6 +106,7 @@ export default function NewOutivityForm({ createOutivity, handleCancel }) {
               id="outivityCountry"
               placeholder="In which country?"
               required
+              defaultValue={isEdit ? outivity.country ?? "" : null}
```

Comment:
same as before ... just a question ;-) 
```suggestion
              defaultValue={isEdit ? outivity.country : null}
```